packages/components/resource-pages/data-resource-list-view.tsx

```diff
@@ -129,11 +130,19 @@ const RowRenderer: React.FC<RowProps<NamespaceStoreKind, CustomData>> = ({
 }) => {
   const { t } = useTranslation();
   const { launchModal, kebabActions, bucketPolicyMap } = rowData;
-  const bucketsCount = bucketPolicyMap[obj?.metadata?.name] || 0;
+  const nameSpaceName = getName(obj);
```

Comment:
```suggestion
  const dataResourceName = getName(obj);
```
we decided all variables will be data-resource and text will be data-source

Reply:
will do, it once all the review comment is fixed.

wanted to have the single details page related commit first then second commit for all name change